tooltester/README.md

```diff
@@ -41,7 +41,7 @@ Follow these setups if there is not a backup
     1. Flavour - c2.large
     1. Source - Ubuntu 18.04 image
     1. Key Pair - JenkinsMaster2
-1. SSH into jump and from there SSH into the slave. Run the setupSlave.sh in master (download from this repository)
+1. SSH into jump and from there SSH into the master and then to slave. Run the setupSlave.sh in the master branch (download from this repository)
```

Comment:
* sounds awkward maybe elaborate e.g. `SSH into the jump server and from there SSH into the master server...`